src/main.js

```diff
@@ -135,6 +135,24 @@ const MyPreset = definePreset(Aura, {
       900: '{red.900}',
       950: '{red.950}',
     },
+    colorScheme: {
+      dark: {
+        surface: {
+          0: 'black',
+          50: '{surface.200}',
+          100: '{surface.300}',
+          200: '{red.200}',
+          300: '{red.300}',
+          400: '{red.400}',
+          500: '{red.500}',
+          600: '{surface.200}',
+          700: '{gray.200}',
+          800: 'white',
+          900: 'white',
+          950: 'white',
+        },
+      },
+    },
```

Comment:
Instead of overriding the `colorScheme`, if we want to enforce light mode we can do this if I remember correctly: 

```diff
app.use(PrimeVue, {
  theme: {
    preset: MyPreset,
+    options: {
+      darkModeSelector: 'dark-mode'
+    }
  },
  ripple: true,
});
``` 

Given the size of the PR, I'd say it would probably best to enforce light mode and use a separate PR (one day down the road) to add full-blown dark mode support if ever required?

Reply:
I can take a look into this, but I believe 'dark-mode' won't enforce light mode as intended. I think this is more for a dark-mode selector. I can try it out tho :)

Reply:
Actually it worked! Thanks!